knowledge/data-sources/notion/credential/tool.gpt

```diff
@@ -0,0 +1,3 @@
+Name: Notion Data Source Credential
+Share Credential: ../../../../oauth2 as notion.sync-file with NOTION_TOKEN as env and notion as integration
+Type: credential
```

Comment:
nit: no newline at the end of this file.

Reply:
Does this mean there is a new line and there shouldn't be? Or visa-versa?

Reply:
there wasn't a new line but should be.